### PR TITLE
Update top_scorers template tag to use score data

### PR DESCRIPTION
**Pull request by phlax**
this is used in the welcome page

### CODE REVIEW

File: pootle/templates/welcome.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
       <div class="section">
         <h1>{% trans 'Top Contributors for the Last 30 Days' %}</h1>
-        {% top_scorers %}
+	{% top_scorers user score_data %}
```
Comment (unho) [inline pootle/templates/welcome.html]: It was using the right indent.

File: pootle/apps/pootle_app/views/index/index.py [hunk 2]
```diff
@@ -7,23 +7,66 @@
...
+            % (self.request.user.username,
+               self.revision,
+               self.request_lang))
+
+    @cached_property
+    def project_set(self):
+        user_projects = Project.accessible_by_user(self.request.user)
+        user_projects = (
+            Project.objects.for_user(self.request.user)
+                           .filter(code__in=user_projects))
```
Comment (unho) [inline pootle/apps/pootle_app/views/index/index.py]: It is not the first time I see this pattern. I do wonder if it is better to move this to a new method in the manager.

Comment (phlax) [inline pootle/apps/pootle_app/views/index/index.py]: not 100% if its right either - but it works 8/

**Comment by unho** [comment]
lgtm

File: pootle/apps/pootle_app/views/index/index.py (hunk 2 above)
Comment (unho) [inline pootle/apps/pootle_app/views/index/index.py]: Yes, it works. Just raising that this looks like repeated code to me.

File: pootle/templates/welcome.html (hunk 1 above)
Comment (phlax) [inline pootle/templates/welcome.html]: it seems not - it should align with the h1

Comment (unho) [inline pootle/templates/welcome.html]: Sure, that was what I meant. Now it is misaligned.

**Pull request merged**